autogpt_platform/frontend/src/app/(platform)/build/components/legacy-builder/CustomNode/CustomNode.tsx

```diff
@@ -158,37 +159,6 @@ export const CustomNode = React.memo(
       setIsAnyModalOpen?.(isModalOpen || isOutputModalOpen);
     }, [isModalOpen, isOutputModalOpen, data, setIsAnyModalOpen]);
 
-    const fillDefaults = useCallback((obj: any, schema: any) => {
```

Comment:
Moved to [@/lib/utils](https://github.com/Significant-Gravitas/AutoGPT/pull/11145/files#diff-db26a69e6fb7546dc621634f3c8ee6efa3639e72e02837f753af18b2fdddf7beR216-R219) and improved